experiments/compositions/composition/internal/controller/expanderversion_controller.go

```diff
@@ -132,7 +132,7 @@ func (r *ExpanderVersionReconciler) processExpanderVersion(
 			value = fmt.Sprintf("%s/expander-%s:%s", ev.Spec.ImageRegistry, expander, key)
 		} else {
 			svcVersion := strings.Replace(key, ".", "-", -1)
-			value = fmt.Sprintf("composition-%s-%s", expander, svcVersion)
+			value = fmt.Sprintf("composition-%s-%s:8443", expander, svcVersion)
```

Comment:
qq: why do we want 8443 here? what does it mean?

Reply:
Its the port where the service listens for connections. 443 is considered ROOT level port.

Reply:
Thank you for the explanation!